portal/src/layout/portal-footer/Cookies.tsx

```diff
@@ -0,0 +1,48 @@
+import { MixpanelBackend } from "@fremtind/jkl-analytics-mixpanel";
+import { useAnalytics } from "@fremtind/jkl-analytics-react";
+import { CookieConsent, useCookieConsent } from "@fremtind/jkl-cookie-consent-react";
+import { WithChildren } from "@fremtind/jkl-core";
+import { graphql, useStaticQuery } from "gatsby";
+import React, { FC, Suspense } from "react";
+
+const mixpanel = new MixpanelBackend({
+    trackingId: process.env.MIXPANEL_PROJECT_ID || "bac3ea28c7d92a05956f54eab1595663",
+});
```

Comment:
Skulle man vurdert å legge til debugging når man kjører lokalt?

```suggestion
const mixpanel = new MixpanelBackend({
    trackingId: process.env.MIXPANEL_PROJECT_ID || "bac3ea28c7d92a05956f54eab1595663",
    debug: process.env.NODE_ENV === "development",
});
```

Reply:
Tenker det kan være opt-in hvis man skulle trenge det. Blir fort støyete om man er i loggene og leter etter noe annet.